sql_scripts/prototype.sql

```diff
@@ -0,0 +1,34 @@
+drop table if exists instruments cascade;
+
+create table instruments as
+  select distinct(instrument) as name from points_prototype;
+
+alter table instruments
+   add column id INT generated always as identity,
+   add primary key(id);
+
+select * from instruments
+
+alter table points_prototype 
+  add instrument_id INT,
+  add constraint fk_instrument
+  foreign key (instrument_id)
+  references instruments (id);
+  
+insert into points_prototype (instrument_id)
+  select instruments.id
+  from instruments 
+  join points_prototype on
+  instruments.name = points_prototype.instrument;
+ 
+ CREATE INDEX instrument_id_idx ON points_prototype (instrument_id);
+ 
```

Comment:
I think we also should create another statement that drops the `instrument` column from the `points_prototype` table. This is to reduce the table size and makes for a more fair performance comparison having to fetch data from disk

Reply:
Yes, I was just thinking about that. I wasn't sure whether to create yet another table, e.g. "points_cleaned" where the unnecessary cols are removed, so we're not having to recreate tables so much?

Reply:
I suggest to do that on the prototype table. Or how long does it take to run this script from start to finish?